src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
-package core.basesyntax;
-
-public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
-
-    }
-}
+package core.basesyntax;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class WorkWithFile {
+    private int supplySum = 0;
+    private int buySum = 0;
+
+    public void getStatistic(String fromFileName, String toFileName) {
+        int result = 0;
+        String data = readFromFile(fromFileName);
+        if (data != null) {
+            result = createReport(data);
+        }
+        if (createNewFile(toFileName)) {
+            writeToFile(toFileName, result);
+        }
+    }
+
+    private boolean createNewFile(String toFileName) {
+        boolean fileCreated;
+        File file = new File(toFileName);
+        try {
+            fileCreated = file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t create file", e);
+        }
+        return fileCreated;
+    }
+
+    private String readFromFile(String fromFileName) {
+        String data;
+        try {
+            data = new String(Files.readAllBytes(Paths.get(fromFileName)));
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t read file", e);
+        }
+        return data;
+    }
+
+    private int createReport(String data) {

```

Comment:
method is named createReport and return type is int
some number cannot be called report